apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -293,6 +299,20 @@ export const blocks = {
       return null;
     };
     generator.gamelab_behavior_get = function () {
+      // If we don't have a behavior Id, find on the definition block.
+      if (!this.behaviorId) {
+        const procedureModelName = this.getProcedureModel().name;
```

Comment:
could `getProcedureModel()` return `null`?

Reply:
`getProcedureModel()` _can_ return `null` if it can't find a model.  This one was a bit hard to test because I couldn't find any case where there wasn't already a behavior id and a model. If we somehow got to this point with neither, I'm not honestly sure how we should attempt to recover.

Reply:
Should we just default to `unknown` in this case? That way we won't throw a null exception here.

Reply:
If there's no model, we could generate the code `'undefined'` similar to further down when return early with that.

Reply:
yep that's what I meant, reviewing too quickly 😬

Reply:
Sounds good - updated!